catalog/urls.py

```diff
@@ -1,10 +1,13 @@
 from django.urls import path
 
+from catalog.apps import CatalogConfig
+
 from . import views
 
-app_name = "catalog"
+app_name = CatalogConfig.name
 
 urlpatterns = [
-    path("home/", views.home, name="home"),
+    path("products_list/", views.products_list, name="products_list"),
+    path("product_detail/<int:pk>", views.product_detail, name="product_detail"),
```

Comment:
В конце нужно добавить слеш, чтобы было одинаково для всех эндпоинтов, причем назвать лучше так:
```suggestion
    path("products/", views.products_list, name="products_list"),
    path("products/<int:pk>/", views.product_detail, name="product_detail"),
```